app/Template/subtask/table.php

```diff
@@ -7,6 +7,7 @@ class="subtasks-table table-striped table-scrolling"
         <tr>
             <th class="column-40"><?= t('Title') ?></th>
             <th><?= t('Assignee') ?></th>
+            <?= $this->hook->render('template:subtask:table:header', array('values' => $values, 'errors' => $errors)) ?>
```

Comment:
again $values and $errors are undefined at this place, replace with 'subtask' => $subtask may be ?